tools/markdown-magic/src/templates/Experimental-Package-Notice-Template.md

```diff
@@ -0,0 +1,2 @@
+**Note: this package is experimental; its APIs may change without notice.**
+**It is not recommended for production scenarios.**
```

Comment:
I usually line break on sentance boundaries in github markdown, since hard line breaks are treated as space-separated in the generated markdown. Makes future git-wise diffs a bit easier to reason about.